README.md

```diff
@@ -234,3 +234,24 @@ Android, iOS, Windows, Tizen
 
 * **Source Code:** [https://github.com/DamianSuess/Learn.MauiPaymentUi]
 * **Author:** Damian Suess - [Blog](https://suesslabs.com/) [GitHub](https://github.com/DamianSuess)
+
+
+## [Bottom Sheets](https://github.com/PremSaiVarada/DemoCustomSheets)
+
+Animated and customised Bottom Sheets plugin for .Net Maui UI [CustomSheets](https://www.nuget.org/packages/CustomSheets)
+
+![Demo Screens](https://user-images.githubusercontent.com/110095923/184805902-a921fb94-f79d-4bec-b160-e4d62ba16b48.png)
```

Comment:
Could you include this image in the images folder from this repository?

Reply:
ok, I will include it

Reply:
added